src/routes/ViewJobLog.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import { Preloader } from '../components/Preloader';
+
+@stripesConnect
+export class ViewJobLog extends Component {
+  static manifest = Object.freeze({
+    logJson: {
+      type: 'okapi',
+      // path: 'source-storage/sourceRecords?query=snapshotId=:{id}',
+      path: 'source-storage/sourceRecords?query=snapshotId=00000000-0000-0000-0000-000000000000&limit=1000',
+      throwsErrors: false,
+    },
+  });
+
+  static propTypes = {
+    resources: PropTypes.shape({
+      logJson: PropTypes.shape({
+        hasLoaded: PropTypes.bool.isRequired,
+        records: PropTypes.arrayOf(
+          PropTypes.object.isRequired
+        ).isRequired,
+      }),
+    }),
+  };
+
+  componentDidMount() {
+    document.querySelector('header').setAttribute('style', 'display: none');
```

Comment:
We need to try to avoid dealing with the plain document when we code with React. The better will be to add CSS styles for `view-job-log-test` container which will stretch it on the whole screen.

Reply:
Thanks for your comments. They are really helpful. But in this case, seems we cannot achieve result using only CSS styles. It is impossible to exceed the z-index of an element while we are a child of a sibling with a smaller z-index.

Reply:
Agree with you. There is no nice way to solve this within *ui-data-import* app. So your approach looks good then.